src/app/content/hooks/locationChange.ts

```diff
@@ -1,22 +1,33 @@
+import css from 'cnx-recipes/styles/output/intro-business.json';
 import { routeHook } from '../../navigation/utils';
 import { receiveBook, receivePage, requestBook, requestPage } from '../actions';
 import { content } from '../routes';
 import * as select from '../selectors';
 import { archiveLoader } from '../utils';
 
-export default routeHook(content, ({dispatch, getState}) => async({match}) => {
+const fontMatches = css.match(/"(https:\/\/fonts\.googleapis\.com\/css\?family=.*?)"/);
+const fonts = fontMatches ? fontMatches.slice(1) : [];
+
+export default routeHook(content, ({dispatch, getState, fontCollector}) => async({match}) => {
```

Comment:
i'm going to make a separate change that makes these hook files export just the hook implementation and move the call to the `routeHook` creator into a more centralized place. it doesn't really matter but it will make the tests more digestible to not maintain knowledge of what `routeHook` is doing